runtime/LibcWrappers.cpp

```diff
@@ -329,6 +376,47 @@ int SYM(fseeko64)(FILE *stream, uint64_t offset, int whence) {
   return result;
 }
 
+struct utmpx *SYM(getutxent)()
```

Comment:
Given that getutxent is an alias for getutent, can you define getutent instead and then alias to it (like [this](https://github.com/eurecom-s3/symcc/commit/9257258e364fcb75533f665f7de1f963435f5146#diff-62f49bb443d58c118ad6bacc805d9e67afb16510ecf130167c73ff767347fb5c))